streams/src/test/java/org/apache/kafka/streams/integration/RegexSourceIntegrationTest.java

```diff
@@ -254,12 +254,14 @@ public void shouldAddStateStoreToRegexDefinedSource() throws Exception {
         streams.start();
 
         final Properties producerConfig = TestUtils.producerConfig(CLUSTER.bootstrapServers(), StringSerializer.class, StringSerializer.class);
+        final Properties consumerConfig = TestUtils.consumerConfig(CLUSTER.bootstrapServers(), StringDeserializer.class, StringDeserializer.class);
+
+        IntegrationTestUtils.produceValuesSynchronously(TOPIC_1, Arrays.asList("testMessage"), producerConfig, mockTime);
+        IntegrationTestUtils.waitUntilMinKeyValueRecordsReceived(consumerConfig, TOPIC_1, 1, 30000);
```

Comment:
nit: we can just call the function with three params which will then use `DEFAULT_TIMEOUT = 30 * 1000L;`.